projects/1.20-fabric/assets/cccbridge/cccbridge/lang/zh_cn.json

```
@@ -0,0 +1,32 @@
+{
+    "block.cccbridge.source_block":"来源方块",

```

```suggestion
    "block.cccbridge.source_block":"源方块",
```
库内1182版本
也可此处不改，更改1182的文件